scripts/configure.py

```diff
@@ -16,10 +16,36 @@
 mission_start_time_utc = datetime.datetime(2000, 1, 1, 12, 0) + datetime.timedelta(seconds=float(mission_start_time))
 print('  start-time-utc:', mission_start_time_utc)
 
+# FSW
+fsw_str = 'fsw'
+fsw_cfg = mission_root.find(fsw_str).text
+print(' ', fsw_str, ':', fsw_cfg)
+fsw_identified = 0
+
+if (fsw_cfg == 'fprime'):
+    fsw_identified = 1
+    os.system('cp ./scripts/fsw_fprime_build.sh ./cfg/build/fsw_build.sh')
+    os.system('cp ./scripts/fsw_fprime_launch.sh ./cfg/build/fsw_launch.sh')
+    os.system('cp ./scripts/fprime.sh ./scripts/docker_launch.sh')
+    os.system('cp ./scripts/fprime_build_fsw.sh ./scripts/docker_build_fsw.sh')
+
+if (fsw_cfg == 'cfs'):
+    fsw_identified = 1
+    # os.system('cp ./scripts/fsw_fprime_build.sh ./cfg/build/fsw_build.sh')
+    # os.system('cp ./scripts/fsw_fprime_launch.sh ./cfg/build/fsw_launch.sh')
+    os.system('cp ./scripts/cfs_cosmos.sh ./scripts/docker_launch.sh')
+    os.system('cp ./scripts/cfs_build_fsw.sh ./scripts/docker_build_fsw.sh')
```

Comment:
Should we overwrite the ./scripts directory or move this into the hidden ~/.nos3 and run it out of there like we do for GSW?

Reply:
making new issue